src/crimson/osd/pg.h

```diff
@@ -159,6 +159,16 @@ class PG : public boost::intrusive_ref_counter<
     bool need_write_epoch,
     ceph::os::Transaction &t) final;
 
+  /// Need to reschedule next scrub. Assuming no change in role
+  void reschedule_scrub() final {
+  }
+
+ /* probably rm */ void on_info_history_change() final {
+    // Not needed yet -- mainly for scrub scheduling
+  }
```

Comment:
https://github.com/ceph/ceph/commit/948e8ee394c374332b3043de09f5b67078a86418 did remove this -- is there a reason you want to reintroduce it?